### PR TITLE
Hero & Navbar

### PR DESCRIPTION
**Pull request by king-11**
## Description

- chore: update config
- chore: text bg color default set
- refactor: use consistent hero everywhere
- feat: footer mt-auto main flex
- feat: new hero with shapes and text
- feat: new navbar to fill space

## Types of changes

- [ ] Bug/Hot fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring Old Code ( non-breaking change replacing already existing code )
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** guidelines in repository.
- [x] My pull request has a descriptive title and is attached to an existing issue.
- [x] I haven't repeated the code and it follows established code style of repository.
- [x] My branch is up-to-date with upstream branch.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by IamEzio** [comment]
Netlify build is failing because `GSAPTimeline` is not defined. I think we can first initialize `openTimeLine` as **this.$gsap.timeline({paused: true})** and then in `configNavbar()` method use
```
openTimelIne.to('.navbar', { left: 0, duration: 0.7 })
.from('.list_item', { duration: 0.4, y: 20, stagger: 0.1, autoAlpha: 0 }, 0.5)
```

**Comment by king-11** [comment]
The reason I see in build is it can't find `$gsap`

**Comment by king-11** [comment]
@IamEzio @Tarun-Arora @somesh202 review your respective pages and other stuff as well

File: components/Home/Hero.vue [hunk 1]
```diff
@@ -1,221 +1,93 @@
...
-      </div>
-    </section>
-  </div>
+      </li>
+    </ul>
+    <figure>
+      <h2 class="text-4xl md:text-5xl lg:text-6xl font-bold">
+        Software Development Group
+      </h2>
+      <figcaption class="text-xl font-medium tracking-widest">Club of Programmers, IIT (BHU) Varanasi</figcaption>
```
Comment (somesh202) [inline components/Home/Hero.vue:16]: The vertical spacing between this and the text below is bit large, can be reduced.

**Review by somesh202 — COMMENTED** [review]
Team page looks good to me.

File: components/navbar.vue [hunk 2]
```diff
@@ -1,5 +1,17 @@
 <template>
-  <nav>
+  <section>
+    <nav class="fixed z-100 w-full backdrop-blur-sm backdrop-filter py-4 hidden md:flex justify-between py-4 px-4 items-center">
+      <aside class="font-bold text-3xl tracking-widest hover:(scale-125) transform px-8 py-2">
+        <nuxt-link to="/">SDG</nuxt-link>
+      </aside>
+      <ul class="ml-auto flex-row md:flex list-none">
+        <li v-for="page in pages" :key="page.url" class="hover:(font-bold scale-125) transform px-8">
```
Comment (IamEzio) [inline components/navbar.vue:8]: A transition of 0.5s could be added -- font size changes suddenly in current version.

**Review by IamEzio — COMMENTED** [review]
The landing page looks really good now. Agree with @somesh202's comment above -- the spacing does look odd on desktop screens. The loading time for home page has also reduced a lot which is awesome.

File: components/navbar.vue (hunk 2 above)
Comment (king-11) [reply components/navbar.vue:8]: Sounds good

File: components/Home/Hero.vue (hunk 1 above)
Comment (king-11) [reply components/Home/Hero.vue:16]: We have to fill the complete screen that's why the spacing is necessary any other alternative for it?

**Review by IamEzio — APPROVED** [review]
LGTM

File: components/Home/Hero.vue (hunk 1 above)
Comment (somesh202) [reply components/Home/Hero.vue:16]: In that case, we can go ahead with this.

**Review by somesh202 — APPROVED** [review]

**Review by S-kus — COMMENTED** [review]
- Carousel buttons for sliding can be of a different color than white.
slider

- **Meet the Team** title is overlapping with the navbar for some screen sizes.
meetTheTeam

**Comment by king-11** [comment]
@S-kus can you make new issues for the same

**Pull request merged**